hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/BucketManagerImpl.java

```diff
@@ -419,7 +421,6 @@ public boolean addAcl(OzoneObj obj, OzoneAcl acl) throws IOException {
     }
```

Comment:
These are legacy addAcl/removeAcl/setAcl before HA is introduced. It is deprecated and not used. 
The actual addAcl/removeAcl/setAcl requests are handled in OMVolumeAclRequest/OMBucketAclRequest/OMKeyAclRequest classes.